### PR TITLE
Modifying the stable/minecraft templates to work with vault secrets

### PR DESCRIPTION
**Pull request by KeenWill**

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -125,6 +125,8 @@ minecraftServer:
   removeOldMods: false
   # Check accounts against Minecraft account service.
   onlineMode: true
+  # Utilize secrets from the Penn Labs Vault
+  secret:
```
Comment (ArmaanT) [inline values.yaml]: You can remove this since we don't want to set a default

**Review by ArmaanT — CHANGES_REQUESTED** [review]

File: values.yaml [hunk 2]
```diff
@@ -125,6 +125,8 @@ minecraftServer:
   removeOldMods: false
   # Check accounts against Minecraft account service.
   onlineMode: true
+  # Utilize secrets from the Penn Labs Vault
```
Comment (KeenWill) [inline values.yaml]:
```suggestion
```

File: values.yaml (hunk 1 above)
Comment (KeenWill) [inline values.yaml]:
```suggestion
```

**Review by KeenWill — COMMENTED** [review]
Fixed

**Review by ArmaanT — APPROVED** [review]
LGTM!

**Pull request merged**